src/pages/graphql/schema/customer/queries/customer.md

```diff
@@ -925,6 +1026,36 @@ import CompareListOutput from '/src/pages/_includes/graphql/compare-list-output.
 
 <CompareListOutput />
 
+### AttributeValueInterface attributes
```

Comment:
Add blank line below

Reply:
there is actually a blank line below. no need for an extra line.